brave-jersey2/src/test/java/com/github/kristofa/brave/jersey2/ITBraveTracingFeature_Server.java

```diff
@@ -59,11 +52,33 @@ public Response disconnect() throws IOException {
     newServer(BraveTracingFeature.builder(brave).spanNameProvider(spanNameProvider).build());
   }
 
+  /**
+   * {@link ContainerResponseFilter} has no means to handle uncaught exceptions. Unless you provide
+   * a catch-all exception mapper, requests that result in unhandled exceptions will leak until they
+   * are eventually flushed.
+   */
+  @Provider
+  public static class ExceptionMapper implements ExtendedExceptionMapper<Exception> {
```

Comment:
we can move this to the jersey2 readme, but I don't think this is safe to add by default as it could interfere with user supplied exception mappers. My guess is that for jersey2, we'd eventually want to use framework-specific code, and this approach only as a workaround. @cschneider @schlosna thoughts?